CHANGELOG.md

```diff
@@ -121,7 +121,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Fixed `CombinedLoader` and `max_size_cycle` didn't receive a `DistributedSampler` ([#10374](https://github.com/PyTorchLightning/pytorch-lightning/issues/10374))
 
 
--
+- Changed `refresh_rate_per_second` param to `referesh_rate` for `RichProgressBar` signature ([#10497](https://github.com/PyTorchLightning/pytorch-lightning/pull/10497))
```

Comment:
If this gets included as a fix I think we should mention the issue. For example:

```suggestion
- Fixed misleading argument name `refresh_rate_per_second` in `RichProgressBar` signature by renaming it to `referesh_rate` ([#10497](https://github.com/PyTorchLightning/pytorch-lightning/pull/10497))
```
or move it to the changed section is also an option

Reply:
Better option! Updated